generate.sh

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+if [[ -z "$TT_TAG" ]]; then
+	echo "Environment variable TT_TAG is not set."
```

Comment:
Why not use `master` by default, if not provided?

Reply:
I want to make it obvious and clear to a user that we generate a code from a release tag or a branch:

* `TT_TAG=master make` for a non-release updates. As example, we want to use `IPROTO_FEATURE_PAGINATION` in a connector code, but there is no releases of Tarantool with the feature yet.
* After a Tarantool release we will make `TT_TAG=2.11.0 make` and create a tag for `go-iproto`.

We need to create a couple of releases to make a stable workflow. But for now, the idea looks like this.